processor/lsmintervalprocessor/internal/merger/model_test.go

```diff
@@ -46,3 +48,171 @@ func TestKeyOrdered(t *testing.T) {
 		before = after
 	}
 }
+
+func TestMergeDeltaExponentialHistogram(t *testing.T) {
```

Comment:
Do we need all of these test cases here? I would think we can defer to the testing of package expo, and just verify here that things are wired up correctly with a basic test.

Reply:
Yup, sound good. Let me remove that!

Reply:
Done!